requirements/requirements.in

```diff
@@ -54,6 +54,36 @@ pycurl==7.45.3
 huggingface-hub==0.25.0
 jinja2==3.1.4
 
+aiohttp==3.10.9
+av==12.3.0
+frozenlist==1.4.1
+fsspec[http]==2024.3.1
+ftfy==6.2.3
+greenlet==3.1.1
+httptools==0.6.1
+importlib-metadata==8.5.0
+importlib-resources==6.4.5
+llvmlite==0.41.1
+markupsafe==2.1.5
+networkx==3.1
+numba==0.58.1
+packaging==24.1
+pkgutil-resolve-name==1.3.10
+pyarrow==17.0.0
+python-dotenv==1.0.1
+regex==2024.9.11
+scikit-learn==1.3.2
+soxr==0.3.7
+termcolor==2.4.0
+tqdm==4.66.5
+transformers==4.41.2
+uvloop==0.20.0
+watchfiles==0.24.0
+websockets==13.1
+zipp==3.20.2
+yarl==1.13.1
```

Comment:
are these all transitive deps? What process did you use to decide which ones should be included?

Reply:
Yes, these are the transitive deps that are not pinned. I run `pip-compile` and compare the output with `pip freeze` on 2.13 and realize the versions of these packages are changed. So I add them in `requirements.in`